suse_openstack_cloud

```diff
@@ -268,6 +268,23 @@ plog_files 0 $crowbar_db_dump_file
 rm -f $crowbar_db_dump_file
 
 #############################################################
+section_header "MariaDB"
+
+find_and_plog_files  /var/log/mysql   -type f
+
+mariadb_variables_file=/tmp/mariadb_variables
+mariadb_status_file=/tmp/mariadb_status
+
+mysql -u monitoring -e 'SHOW VARIABLES;' > $mariadb_variables_file
```

Comment:
These lines should not get run if `mysql` isn't installed, otherwise it will cause errors when trying to run a non-existent program.